wrappers/rust/icicle-core/Cargo.toml

```diff
@@ -2,21 +2,24 @@
 name = "icicle-core"
 version = "1.0.0"
 edition = "2021"
-authors = [ "Ingonyama" ]
+authors = ["Ingonyama"]
 description = "A library for GPU ZK acceleration by Ingonyama"
 homepage = "https://www.ingonyama.com"
 repository = "https://github.com/ingonyama-zk/icicle"
 
 
-[dependencies] 
+[dependencies]
 
 icicle-cuda-runtime = { path = "../icicle-cuda-runtime" }
 
 ark-ff = { version = "0.4.0", optional = true }
-ark-ec = { version = "0.4.0", optional = true, features = [ "parallel" ] }
+ark-ec = { version = "0.4.0", optional = true, features = ["parallel"] }
 ark-poly = { version = "0.4.0", optional = true }
 ark-std = { version = "0.4.0", optional = true }
 
+#[dev-dependencies] # TODO: add as dev dependency?
```

Comment:
Whats stopping us from making this a dev-dependency now?

Reply:
I guess since test files are importing files containing `check_...` functions - these are available not only in `#[test]` - maybe this for examples? I personally would add rayon not only to tests - since it's imo the defacto standard for (non-async) multithreading anyway. but maybe we would like go async or tokio way etc

Reply:
just double checked - it builds if rayon added as dev-dependency, probably rust-analyser issue